package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "JSONCrush",
+  "version": "1.0.1",
+  "description": "Compress JSON in URL friendly strings",
+  "main": "dist/JSONCrush.cjs.js",
+  "module": "dist/JSONCrush.esm.js",
+  "browser": "dist/JSONCrush.umd.js",
+  "types": "JSONCrush.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "rollup -c",
+    "preinstall": "npm install --ignore-scripts && npm run build"
```

Comment:
The bundle should be built when [`npm publish`](https://docs.npmjs.com/cli/v7/using-npm/scripts#npm-publish) (or the yarn equivalent) is run. I suggest doing in `scripts.prepare`.